mediaflow_proxy/utils/packed.py

```
@@ -0,0 +1,159 @@
+        soup = BeautifulSoup(response.text, "lxml",parse_only=SoupStrainer("script"))
+        script_all = soup.find_all("script")
+        for i in script_all:
+            if detect(i.text):
+                unpacked_code = unpack(i.text)
+                match = re.search( r'file:"(.*?)"', unpacked_code)
+                if match:
+                    m3u8_url = match.group(1)
+                    return m3u8_url
+    except Exception as e:
+        logger.error("Eval solver error\n",e)
+        raise Exception("Error in eval_solver")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)